lms/static/sass/views/_login-register.scss

```diff
@@ -509,16 +509,16 @@
         }
 
         &.button-oa2-azuread-oauth2 {
-            color: darken($microsoft-blue, 20%);
+            color: darken($microsoft-black, 20%);
```

Comment:
I don't think this will make a darker color than black. You may need a slightly lighter version.